massa-node/base_config/initial_vesting.json

```diff
@@ -0,0 +1,22 @@
+{
+    "Address": [
+        {
+            "slot": {
+                "start_period": 0,
+                "end_period": 99,
+                "thread": 0
```

Comment:
A slot is a `period` and a `thread` you can't have this 3 fields in your slot. I think it's better to have a `start_slot` and `end_slot` and both have `period` and `thread`.